Sources/BraintreeThreeDSecure/V2UICustomization/BTThreeDSecureV2BaseCustomization.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+import CardinalMobile
+
+/// Base customization options for 3D Secure 2 flows.
+@objcMembers public class BTThreeDSecureV2BaseCustomization: NSObject {
+
+    // MARK: - Public Properties
+
+    /// Font type for the UI element.
+    public var textFontName: String? {
+        get { _textFontName }
+        @objc(setTextFontName:) set {
+            _textFontName = newValue
+            cardinalValue?.textFontName = newValue
+        }
+    }
+
+    /// Color code in Hex format. For example, the color code can be “#999999”.
+    public var textColor: String? {
+        get { _textColor }
+        @objc(setTextColor:) set {
+            _textColor = newValue
+            cardinalValue?.textColor = newValue
+        }
+    }
+
+    /// Font size for the UI element.
+    public var textFontSize: Int {
+        get { _textFontSize }
+        @objc(setTextFontSize:) set {
+            _textFontSize = newValue
+            cardinalValue?.textFontSize = Int32(newValue)
+        }
+    }
+
+    // MARK: - Internal Properties
+
+    var cardinalValue: Customization?
+
+    // MARK: - Fileprivate Properties
+
+    /// Used as a holder for Obj-C interoperability
+    fileprivate var _textFontName: String?
```

Comment:
random q: Can `private` be used here?

Reply:
Yeah we can use either - if we have a preference I can update these to private. I guess it's unlikely something in this file needs access to these properties.